Tools/Scripts/webkitpy/benchmark_runner/http_server_driver/http_server/signposts/setup.py

```diff
@@ -0,0 +1,13 @@
+from distutils.core import setup, Extension
+
+module = Extension(
+    'signposts',
```

Comment:
I think we should name this `browsersignpostspy`.

We should move it to `Tools/Scripts/libraries/browsersignpostspy`, I'll note on line `9` exactly how I envision the `AutoInstall` integration working.